src/templates/finops-hub/modules/dataFactory.bicep

```diff
@@ -168,15 +178,49 @@ resource stopHubTriggers 'Microsoft.Resources/deploymentScripts@2020-10-01' = {
 // Linked services
 //------------------------------------------------------------------------------
 
+resource linkedService_keyVault 'Microsoft.DataFactory/factories/linkedservices@2018-06-01' = {
+  name: keyVaultName
+  parent: dataFactory
+  properties: {
+    annotations: []
+    parameters: {}
+    type: 'AzureKeyVault'
+    typeProperties: {
+      baseUrl: 'https://${keyVaultName}${environment().suffixes.keyvaultDns}/'
```

Comment:
The template analyzer used by AQT doesn't like when we generate URLs. Can we get this as a property from the KeyVault instance? If not, maybe leave a comment that we can't get it from properties and we have to generate it.